tests/scripts/gen_release_notes.sh

```diff
@@ -16,27 +16,27 @@ if [ -z "${GITHUB_USER}" ] || [ -z "${GITHUB_TOKEN}" ]; then
   help
 fi
 
-pr_list=$(git log --pretty="%s" --merges --left-only ${FROM_BRANCH}...${TO_TAG} | grep pull | awk '/Merge pull request/ {print $4}' | cut -c 2-)
+pr_list=$(git log --pretty="%s" --merges --left-only "${FROM_BRANCH}"..."${TO_TAG}" | grep pull | awk '/Merge pull request/ {print $4}' | cut -c 2-)
 
 # for releases notes
 function release_notes() {
   for pr in $pr_list; do
   # get PR title
-  backport_pr=$(curl -s -u ${GITHUB_USER}:${GITHUB_TOKEN} https://api.github.com/repos/rook/rook/pulls/${pr} | jq '.title')
+  backport_pr=$(curl -s -u "${GITHUB_USER}":"${GITHUB_TOKEN}" https://api.github.com/repos/rook/rook/pulls/"${pr}" | jq '.title')
```

Comment:
I would put the whole URL in quotes to be safe. I'm not sure how curl handles a URL like https://api.github.com/blah/"my thing"

```suggestion
  backport_pr=$(curl -s -u "${GITHUB_USER}":"${GITHUB_TOKEN}" "https://api.github.com/repos/rook/rook/pulls/${pr}" | jq '.title')
```